vpr/src/route/connection_router.h

```diff
@@ -46,8 +45,13 @@ class ConnectionRouter : public ConnectionRouterInterface {
         , router_stats_(nullptr)
         , router_debug_(false) {
         heap_.init_heap(grid);
-        heap_.set_prune_limit(rr_nodes_.size(), kHeapPruneFactor * rr_nodes_.size());
         only_opin_inter_layer = (grid.get_num_layers() > 1) && inter_layer_connections_limited_to_opin(*rr_graph);
+        rr_node_R_upstream.resize(rr_node_route_inf.size());
+        rcv_path_data.resize(rr_node_route_inf.size());
+    }
+
+    ~ConnectionRouter() {
+        VTR_LOG("Serial Connection Router is being destroyed. Time spent computing SSSP: %.3f seconds\n.", this->sssp_total_time.count() / 1000000.0);
```

Comment:
@ueqri I think this line is the cause for the failures you are seeing in the CI.

You wrote:
```
"Serial Connection Router is being destroyed. Time spent computing SSSP: %.3f seconds\n."
```

I think you meant:
```
"Serial Connection Router is being destroyed. Time spent computing SSSP: %.3f seconds.\n"
```
(Notice you put the period AFTER the return carriage).

I was reviewing the logs of the failed testscases and noticed:
<img width="872" alt="image" src="https://github.com/user-attachments/assets/e18f7905-9d2c-4622-9cc3-1647154ceaf4">

I wonder if this period on the following line could be causing this issue. Regardless this should be fixed.

Reply:
Good eyes @AlexandreSinger !